docs/dashboard/output-variables.md

```diff
@@ -0,0 +1,50 @@
+<!--
+title: Serverless Dashboard - Output Variables
+menuText: Output Variables
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/)
```

Comment:
Docs site link needs to be updated to match the link for this page in docs/README.md